lib/matplotlib/patches.py

```diff
@@ -414,7 +414,7 @@ def set_linestyle(self, ls):
 
         Alternatively a dash tuple of the following form can be provided::
 
-            (offset, onoffseq),
```

Comment:
This was at least misleading. In python syntax the trailing comma wraps the `(offset, onoffseq)` in an extra 1-element tuple.